index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+    <main id = "main">
+     <h1 id = "title">
+       Dr. Norman Borlaug
+     </h1>
+     <div id= "img-div">
+       <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
```

Comment:
I noticed that you did not write an alt attribute for your image. This attribute provides a description of an image in case the image cannot be viewed due to a slow internet connection or an error in the src attribute. For good practices, please write a descriptive alt attribute.

Reply:
Thank you for your review and positive feedback. 👍  I appreciate your encouragement. It is a really good review comment. 💯